eda/tests/unittests/analysis/test_interaction.py

```diff
@@ -1,12 +1,34 @@
 from unittest.mock import MagicMock
 
 import pandas as pd
+import numpy as np
 import pytest
+import os
```

Comment:
Not a big deal, but import order is weird here

Reply:
BTW, just realized that lint check are not enabled for EDA. Is this intended? I somehow remember you've enabled it @gradientsky

Reply:
there is a lint check: https://github.com/autogluon/autogluon/blob/master/.github/workflow_scripts/test_eda.sh#L14